packages/react/src/themes/base/components/Grid/gridStyles.ts

```diff
@@ -24,6 +24,7 @@ const gridStyles: ComponentSlotStylesInput<GridProps, GridVariables> = {
       gridGap,
       display: 'grid',
       justifyContent: 'space-evenly',
+      '& > *': { outlineOffset: '-3px' },
```

Comment:
as we've discussed, I am suggesting to introduce this as part of the Teams theme, as this is about Teams scenarios that have introduced this issue. For other scenarios we would just introduce the same behavior (and visual appearance) as plain browser styles would provide, so I am suggesting to stick to that base browser's one in the `base` theme

Reply:
oops...didn't notice it was moved to base; fixed, thanks!